### DISCUSSION TITLE
[Actions] Windows系统在build时出错

### DISCUSSION THREAD

**Discussion by realnoob007** [tw93/Pake#523] [Q&A]
求助! 按照wiki中写的developer method一步步操作的，确认所有ico, png的生成，csv的填写都没问题
<img width="1008" alt="微信截图_20230720202734" src="https://github.com/tw93/Pake/assets/87698941/8a0704ea-66dc-478a-b211-cce0886a5887">

**Comment by tw93** [comment]
https://tauri.app/v1/guides/getting-started/prerequisites/
有参考这个吗配置环境吗
